### DISCUSSION TITLE
Using ChatGPT-wrapper in non-GUI OS

### DISCUSSION THREAD

**Discussion by DesFelasaf** [llm-workflow-engine/llm-workflow-engine#78] [Q&A]
Hi! Just set up your wrapper on my local - it's AMAZING! Thank you!
It opens so much opportunities for my pet project - chat bot with AI integration
I want to run bot-script on my dedicated server, but it has no GUI OS (Ubuntu Server).
But as I've noticed your lib needs to start a browser session in order to authenticate in OpenAPI.
I've not tried it yet by myself and decided to ask here - will it run on non-GUI OS such as Ubuntu Server?

**Comment by thehunmonkgroup** [comment]
API backend should be able to run w/ no GUI.
